### DISCUSSION TITLE
Default language

### DISCUSSION THREAD

**Discussion by JairGuzman01** [jdepoix/youtube-transcript-api#177] [Q&A]
Hello,

Is there any way to get the default language of a video?

For example, a video has several captions, but I want the captions of the language of the video... if the video is in Spanish, I want to get those captions…

Thanks!

**Comment by jdepoix** [comment]
Hi @JairGuzman01,

unfortunately, this is currently not supported. Adding this feature has already been discussed in #133. Although this would be somewhat of a breaking chance, I would not be opposed to making the module use the default language of the video instead of english. I haven't gotten around to implementing this yet, but I am open to contributions on this 😊
